eng/build-job.yml

```diff
@@ -72,9 +72,19 @@ jobs:
         value: $(buildConfigUpper)
     - name: officialBuildIdArg
       value: ''
+    - name: ibcOptimizeArg
+      value: ''
     - ${{ if and(eq(variables['System.TeamProject'], 'internal'), ne(variables['Build.Reason'], 'PullRequest')) }}:
       - name: officialBuildIdArg
         value: '-officialbuildid=$(Build.BuildNumber)'
+      - name: ibcOptimizeArg
+        value: '-ibcoptimize'
```

Comment:
This may need to be Windows only unless you want to hook up the legs for optimizing linux, which requires the ibcoptimize step to happen on Windows, since IBCMerge currently is not crossplat (I can't tell if this is Windows only or not from context).

Basically what you would do is skip crossgen on the initial build, copy the artifacts to a windows job, run build.cmd -linuxmscorlib -release -ibconly, then bring it back over to linux and run build.sh -release -crossgenonly -partialngen (see here: https://github.com/dotnet/coreclr/pull/20496#issuecomment-431504174)

Reply:
It's also only available for x64 and x86, not arm.

Reply:
I didn't realize that IBCMerge was Windows-only. I think I'm going to only enable it on Windows for this PR (to bring parity to the previous build system). The change to support ibc optimizations for Linux builds is going to be a large refactoring, so I'm going to do that in a different PR.